spec.html

```diff
@@ -21182,6 +21180,23 @@ <h1>Runtime Semantics: ClassDefinitionEvaluation</h1>
           1. Perform _classScope_.InitializeBinding(_classBinding_, _F_).
         1. Return _F_.
       </emu-alg>
+
+      <emu-clause id="sec-default-constructor-functions">
+        <h1>Default Constructor Functions</h1>
+        <p>When a Default Constructor Function is called with zero or more arguments which form the rest parameter ..._args_, the following steps are taken:</p>
+        <emu-alg>
+          1. If NewTarget is *undefined*, throw a *TypeError* exception.
```

Comment:
This should be changed so that the `TypeError` is created in the caller realm, so that there’s no observable difference between `(class {})()` and `(class { constructor() {} })()`.

Reply:
thanks for reminding me, we got consensus to change the error to always be in the callee realm instead of the caller realm.